src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -117,6 +117,14 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.CreationTimestamp" Header="Created" Width="138"  MinWidth="100">
+                    <DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <TextBlock Text="{Binding TaintIssueViz.Issue.CreationTimestamp, StringFormat=G}" 
```

Comment:
Display in general local date-and-time format. I considered just showing the date, but decided on date and time. We can tweak the format if necessary.

Reply:
I think the most polished would be the user-friendly texts, e.g. "6 months ago". I don't know if there's any built-in way to do that. We can tweak it later if needed.